draft-perrig-scion-overview.md

```diff
@@ -165,24 +167,24 @@ SCION operates on two routing levels: intra-ISD and inter-ISD. As a path-based a
 
 The process of creating an end-to-end forwarding path consists of the following steps:
 
-1. First, an AS discovers paths to other ASes, during the *path exploration* (or beaconing) phase.  
-2. The AS then selects a few PCBs according to defined selection criteria, transforms the selected PCBs into path segments, and registers these segments with a path infrastructure, thus making them available to other ASes. This happens during the *path registration* phase.  
-3. During the *path resolution* phase, the actual creation of an end-to-end forwarding path to the destination takes place. For this, an end host performs  
-      a. a *path lookup* step, to obtain path segments, and  
-      b. a *path combination* step, to combine the forwarding path from the segments.  
+1. First, an AS discovers paths to other ASes, during the *path exploration* (or beaconing) phase.
+2. The AS then selects a few PCBs according to defined selection criteria, transforms the selected PCBs into path segments, and registers these segments with a path infrastructure, thus making them available to other ASes. This happens during the *path registration* phase.
+3. During the *path resolution* phase, the actual creation of an end-to-end forwarding path to the destination takes place. For this, an end host performs
+      a. a *path lookup* step, to obtain path segments, and
+      b. a *path combination* step, to combine the forwarding path from the segments.
 
-**ISD and AS numbering**  
+**ISD and AS numbering**
 Routing is based on the <ISD, AS> tuple, agnostic of local addressing. Existing AS numbers are inherited from the current Internet, but a 48-bit namespace allows for additional SCION AS numbers beyond the 32-bit space in use today. Host addressing extends the network address with a local address, forming the <ISD, AS, local address> 3-tuple. The local address is not used in inter-domain routing or forwarding, does not need to be globally unique, and can thus be an IPv4, IPv6, or MAC address, for example.
 
 ### Infrastructure Components
 
 The **beacon service**, the **path service**, and the **certificate service** are the main infrastructure components of the SCION network architecture. Each AS can have one or more servers per service, depending on the AS's size and type. It is also possible to combine the services into one or more *control services*. *Internal routers* forward packets inside the AS, while *border routers* provide interconnectivity between ASes.
```

Comment:
> It is also possible to combine the services into one or more *control services*. 

What do you mean? Those services are already control services, so I'm a bit confused by this sentence. Perhaps you mean that they can be deployed on the same machine?